vermouth/processors/average_beads.py

```diff
@@ -88,7 +88,7 @@ def do_average_bead(molecule, ignore_missing_graphs=False, weight=None):
                 if subnode.get('position') is not None
             ])
             if abs(sum(weights)) < 1e-7:
-                node['position'] = np.nan
+                node['position'] = np.array([np.nan]*positions.shape[1], dtype=float)
```

Comment:
`np.full_like(positions, np.nan, dtype=float)` ?

Reply:
No, because positions is 2D, and node['position'] should be 1D